### ISSUE TITLE
On the hyper-parameter searching and test model selection

### ISSUE THREAD

**Issue by YBZh** [thuml/Transfer-Learning-Library#47]
Thanks for your excellent contributions !! I have two questions:
1. How do you set the hyper-parameters (e.g., lr, lr-gamma, and trade-off')? Are these hyper-parameters set according to the performance of the test data? 
2. It seems that the best test results in the whole training process are reported. In other words, you also select the test model according to the performance of the test data, right?

Looking forward to your reply.

**Comment by microhhh** [comment]
1. The hyper-parameters, such as lr-gamma, and trade-off, are generally based on the report of the papers. Due to historical problems, they adjusted the parameters on the test set.
2. The best test results in the whole training process are reported. This is the case in the implementation of some papers. In order to make a fair comparison, we also temporarily use this method. In fact, we should report the results of a fixed number of iterations.

**Comment by YBZh** [comment]
Got it. Thanks for your reply.
